ts/protocol/utils.ts

```diff
@@ -132,7 +132,9 @@ export function stringValidator(value: any): boolean {
  * Returns true if the current origin is https.
  */
 export function isSecureOrigin(): boolean {
-  return window.location.protocol === 'https:';
+  return window.location.protocol === 'https:' ||
```

Comment:
Thanks. I totally agree, but it takes time to get these PRs in and incorporate them in our code, so for such a small change I assumed it'd be fine.